packages/ckeditor5-build-balloon-block/tsconfig.release.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "../../tsconfig.release.json",
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": true,
```

Comment:
```suggestion
```

I'd move control over typings to the root config.

https://github.com/ckeditor/ckeditor5/blob/bb0f347c09668e6a276dbe374f705db38a87b6b3/tsconfig.release.json#L5

Thanks to that, we need to update a single field to emit typing everywhere.